mathsys/v5/builder/code.py

```
@@ -0,0 +1,91 @@
+                "cargo",
+                "+nightly",
+                "build",
+                *(["--release"] if optimize else []),
+                "--target", TARGETS[target]
+            ], environment)
+            if optimize: 
+                for command in self.post(target):
+                    try: self.execute(command, environment)
+                    except: pass
+            with open(self.filename(target), "rb") as file: return file.read()
+        except: raise
```

Ruff (0.14.8)

Remove exception handler; error is immediately re-raised

(TRY203)